srcs/web/WebhookManager.py

```diff
@@ -91,6 +91,17 @@ def workflow_job(self, payload: WebHook):
                 "labels": payload.workflow_job.labels,
             }
         )
+
+        runner_m.factory.cloud_manager.add_labels_to_instance(
+            payload.workflow_job.runner_name,
+            dict(
+                status=status["status"],
+                repository=payload.repository.name,
+                workflow=payload.workflow_job.workflow_name,
```

Comment:
Wait a minute, we now have the workflow name in the workflow job webhook? That's some good news, something we can use in the exporter to calculate costs of workflows instead of jobs.

Reply:
Yeahhhh!!! A ticket has been create for this for the next sprint